apps/ws/package.json

```diff
@@ -31,7 +31,7 @@
     "@nestjs/platform-socket.io": "^9.2.1",
     "@nestjs/swagger": "^4.8.0",
     "@nestjs/terminus": "^7.2.0",
-    "@nestjs/websockets": "^7.6.17",
```

Comment:
I think this was the guilt part for the error message for the users. Couldn't reproduce it though.
It should be the same version as `@nestjs/platform-socket.io`.